### PR TITLE
Fix usaco 992

### PR DESCRIPTION
**Pull request by Aadit-Ambadkar**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/cf-1702E.mdx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    
+	for (pair<int, int>& i : dm) {
+		cin >> i.first >> i.second;
+	}
+		
+	for (pair<int, int> i : dm) {
+        	// Check if a dominoe has the same number twice
+		if (i.first == i.second) {
+			cout << "NO\n";
+			return;
```
Comment (SansPapyrus683) [inline solutions/cf-1702E.mdx:76]:
```suggestion
			cout << "NO" << endl;
			return 0;
```

File: solutions/cf-1702E.mdx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+		}
+	}
+
+	map<int, vector<int>> ma;
```
Comment (SansPapyrus683) [inline solutions/cf-1702E.mdx:80]: what's `ma`

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/usaco-992.mdx [hunk 3]
```diff
@@ -9,8 +9,12 @@ author: Óscar Garries, Neo Wang
 
 [Official Analysis](http://www.usaco.org/current/data/sol_wormsort_silver_jan20.html)
 
+**Time Complexity:** $\mathcal{O}(N\log M)$ <!-- Not quite sure if the variables are right.-->
```
Comment (bqi343) [inline solutions/usaco-992.mdx:12]: should be $\mathcal O((N+M)\log \max w_i)$

Comment (Aadit-Ambadkar) [reply solutions/usaco-992.mdx:12]: Thanks!

**Comment by Aadit-Ambadkar** [comment]
Ahhh there were some issues with this pull request so i'm closing it. I'll make another one rq.

**Pull request closed without merging**